### PR TITLE
FE keep alive mechanism for SSOe session

### PR DESCRIPTION
**Pull request by hinzed1127**

### CODE REVIEW

File: src/platform/utilities/api/index.js [hunk 1]
```diff
@@ -18,6 +33,7 @@ export function fetchAndUpdateSessionExpiration(...args) {
         if (sessionExpiration) {
           localStorage.setItem('sessionExpiration', sessionExpiration);
         }
+        refreshSSOeSession();
```
Comment (hinzed1127) [inline src/platform/utilities/api/index.js]: this could go elsewhere, but it seemed to make sense to group it with the current `vets-api` logic

**Review by bshyong — APPROVED** [review]

File: src/platform/utilities/api/index.js [hunk 2]
```diff
@@ -3,6 +3,21 @@ import * as Sentry from '@sentry/browser';
 import environment from '../environment';
 import localStorage from '../storage/localStorage';
 
+function refreshSSOeSession() {
+  // TODO replace arbitrary resource fetch with more intentional enpoint
+  fetch('https://eauth.va.gov/favicon.ico', {
```
Comment (patrickvinograd) [inline src/platform/utilities/api/index.js]: This probably needs to be parameterized per-environment, so dev goes to int.eauth.va.gov, etc.

Comment (hinzed1127) [reply src/platform/utilities/api/index.js]: adding environment mappings. there's also `http://preprod.eauth.va.gov/`, but I don't think there's an analogue for that on our side

**Comment by hinzed1127** [comment]
Outdated, will reopen after resolving ongoing CORS issues with a specific `keepalive` IAM endpoint

**Pull request closed without merging**